builtins_benchmark_test.go

```diff
@@ -0,0 +1,19 @@
+package jsonnet
+
+import (
+	// "os"
+	"os/exec"
+	"testing"
+)
+
+func Benchmark_Builtin_substr(b *testing.B) {
+	for n := 0; n < b.N; n++ {
+		cmd := exec.Command("./jsonnet", "./builtin-benchmarks/substr.jsonnet")
+		// cmd.Stdout = os.Stdout
```

Comment:
Have you left these lines commented out intentionally?

Reply:
yes, they are useful for debugging. In the future I might decide on a better way.

I'm not even sure if this is the right way to do this testing. If you have any suggestions, I'm open! :)

Reply:
I think it's ok for now as a first step. I was running `jsonnet` command by hand for this purpose. 

If we're going to use Go benchmarks for that, we probably should allow passing the path to Jsonnet command by commandline. The output redirection thing could also be a flag,

Reply:
I agree. The way I did it worked, but was tedious.

1. comment out the line in the `builtins.go` file that overwrote the upstream jsonnet implementation
2. build the jsonnnet binary
3. run the benchmark
4. uncomment the line in `builtins.go`
5. build the binary
6. run the benchmark

Reply:
I agree. The way I did it worked, but was tedious.

1. comment out the line in the `builtins.go` file that overwrote the upstream jsonnet implementation
2. build the jsonnnet binary
3. run the benchmark
4. uncomment the line in `builtins.go`
5. build the binary
6. run the benchmark